tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_replay_cache.c

```diff
@@ -79,6 +80,7 @@ static void rx_ended(uint8_t *data, size_t len)
 
 static void test_tx_immediate_replay_attack(void)
 {
+	stored_keys_clear();
```

Comment:
Actually no because some tests within test suite require them some not. There is no common initialization for them. It is exactly how we emulate restart with stored persistent data.